docs/pages/management/dynamic-resources/terraform-provider.mdx

```diff
@@ -83,66 +81,61 @@ To prepare credentials for a local Teleport user:
            - access_list
            verbs: ['list','create','read','update','delete']
    version: v7
-   ---
-   kind: user
-   metadata:
-     name: terraform
-   spec:
-     roles: ['terraform']
-   version: v2
    ```
 
-   These settings configure a user and role named `terraform` with the permissions
+   These settings configure a role named `terraform` with the permissions
    required to manage resources in your Teleport cluster.
 
-1. Create the `terraform` user and role by running the following command:
+1. Create the `terraform` role by running the following command:
 
    ```code
-   $ tctl create -f terraform.yaml
+   $ tctl create -f terraform_role.yaml
    role 'terraform' has been created
-   user "terraform" has been created
    ```
 
-   The `terraform` user can't sign in to get credentials, so you must have another user
-   **impersonate** the `terraform` account to request a certificate.
+1. Create a new file called `create_and_run_terraform_bot.sh`, open it in an editor, and
+paste in the following content:
 
-1. Create a new file called `terraform-impersonator.yaml` and open it in an editor.
+   ```shell
+   #! /usr/bin/env bash
 
-1. Configure a role that enables your user to impersonate the Terraform user by pasting
-the following content into the `terraform-impersonator.yaml` file:
+   set -euxo pipefail
 
-   ```yaml
-   kind: role
-   version: v7
-   metadata:
-     name: terraform-impersonator
-   spec:
-     allow:
-       # This impersonate role allows any user assigned to this role to impersonate
-       # and generate certificates for the user named "terraform" with a role also
-       # named "terraform".
-       impersonate:
-         users: ['terraform']
-         roles: ['terraform']
+   TOKEN=$(tctl bots add terraform-test --roles=terraform --format=json | jq -r '.token_id')
```

Comment:
Side note: `tctl tokens add` has a `--format=token` option that just outputs the raw token, making `jq` unnecessary here. We should probably do the same for `tctl bots add` at some point.

Reply:
@zmb3 I got this when trying to use `--format=token`

```
ERROR: enum value must be one of text,json, got 'token'
```

Reply:
Oh you're saying `tokens add`, this is `bots add`. I've been through the rigamorole that ended in "you can't create a token and then give it to the bot as a value.